checks/libs/win/winpdh.py

```diff
@@ -0,0 +1,49 @@
+import time
+import sys
+import win32pdh
+import psutil
+
+class WinPDHCounter(object):
+    
+    def __init__(self, class_name, instance_name):
+        self._class_name = class_name
+        self._instance_name = instance_name
```

Comment:
I did it this way to match as closely as possible the previous calling convention.  I'd be happy to make that change if it's helpful.